cmd/feed-ingress/main.go

```diff
@@ -4,25 +4,82 @@ import (
 	"os"
 	"os/signal"
 	//
+	"flag"
+
+	"io/ioutil"
+
 	log "github.com/Sirupsen/logrus"
 	"github.com/sky-uk/feed/ingress"
 	"github.com/sky-uk/feed/k8s"
 )
 
+var (
+	apiServer  string
+	caCertFile string
+	tokenFile  string
+	debug      bool
+)
+
+func init() {
+	const (
+		defaultAPIServer  = "https://10.254.0.1:443"
```

Comment:
I don't think we should put a default in here in case we deploy it to one cluster and it is pointing to anther. Maybe failing to start is better?